catalyst-gateway/bin/src/db/index/schema/cql/unstaked_txo_assets_by_txn_hash.cql

```diff
@@ -3,8 +3,8 @@ CREATE TABLE IF NOT EXISTS unstaked_txo_assets_by_txn_hash (
     -- Primary Key Fields
     txn_hash        blob,     -- 32 byte hash of this transaction.
     txo             smallint, -- offset in the txo list of the transaction the txo is in.
-    policy_id       blob,     -- asset policy hash (id) (28 byte binary hash)
-    policy_name     text,     -- name of the policy (UTF8)
+    asset_id        blob,     -- asset policy hash (id) (28 byte binary hash)
+    asset_name      text,     -- name of the policy (UTF8) (32 bytes)
```

Comment:
Same comment for `asset_id` and `asset_name` as discussed above.
Change `asset_is` back to `policy_id` change the type of `asset_name` to be a blob and treat it as a Vec<u8> in the code, never a string.